Userland/Libraries/LibGUI/FileSystemModel.cpp

```diff
@@ -122,8 +122,19 @@ void FileSystemModel::Node::traverse_if_needed()
         total_size += child->size;
         if (S_ISDIR(child->mode))
             directory_children.append(move(child));
-        else
-            file_children.append(move(child));
+        else {
```

Comment:
when adding braces to the else here, please add them to the corresponding `if` above to keep the entire block consistent.

Reply:
This and the below has been fixed, should it be a new commit or amended into the older commits and then force pushed?

Reply:
Sorry, looks like I missed this comment. The fixed formatting should be fixed-up into the commit that introduced the formatting mishap.